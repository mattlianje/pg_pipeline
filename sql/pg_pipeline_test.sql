-- Test that extension can be created
CREATE EXTENSION pg_pipeline;

-- Test the hello world function
SELECT pipeline_hello('PostgreSQL');
SELECT pipeline_hello(NULL);

-- Clean up
DROP EXTENSION pgpipeline;
