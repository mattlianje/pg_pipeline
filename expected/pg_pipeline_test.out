-- Test that extension can be created
CREATE EXTENSION pg_pipeline;
-- Test the hello world function
SELECT pipeline_hello('PostgreSQL');
   pipeline_hello   
--------------------
 Hello, PostgreSQL!
(1 row)

SELECT pipeline_hello(NULL);
  pipeline_hello  
------------------
 Hello, world!
(1 row)

-- Clean up
DROP EXTENSION pg_pipeline;
